addons/l10n_nz/data/account_tax_report_data.xml

```diff
@@ -0,0 +1,107 @@
+        <field name="name">[BOX 9] Enter any adjustments from your calculation sheet</field>
+        <field name="tag_name">BOX 9</field>
+        <field name="parent_id" ref="tax_report_sale_and_income"/>
+        <field name="sequence" eval="5"/>
+        <field name="code">BOX9</field>
+        <field name="formula"></field>
+        <field name="country_id" ref="base.nz"/>
+    </record>
+
+    <record id="tax_report_box10" model="account.tax.report.line">
+        <field name="name">[BOX 10] Total GST collected on sales and income</field>
+        <field name="parent_id" ref="tax_report_sale_and_income"/>
+        <field name="sequence" eval="6"/>
+        <field name="formula">(((BOX5 - BOX6) * 3)/23) + BOX9</field>
+        <field name="country_id" ref="base.nz"/>
+    </record>
+
+    <record id="tax_report_purchases_and_expenses" model="account.tax.report.line">
+        <field name="name">Purchases and Expenses</field>
+        <field name="sequence" eval="2"/>
+        <field name="country_id" ref="base.nz"/>
+    </record>
+
+    <record id="tax_report_box11" model="account.tax.report.line">
+        <field name="name">[BOX 11] Total purchases and expenses(including GST) for which tax invoicing requirements have been met</field>
+        <field name="tag_name">BOX 11</field>
+        <field name="parent_id" ref="tax_report_purchases_and_expenses"/>
+        <field name="sequence" eval="1"/>
+        <field name="code">BOX11</field>
+        <field name="country_id" ref="base.nz"/>
+    </record>
+
+    <record id="tax_report_box12" model="account.tax.report.line">
+        <field name="name">[BOX 12] Multiply BOX11 by 3 and then divide by 23</field>
+        <field name="parent_id" ref="tax_report_purchases_and_expenses"/>
+        <field name="sequence" eval="2"/>
+        <field name="formula">(BOX11 * 3)/23</field>
+        <field name="country_id" ref="base.nz"/>
+    </record>
+
+    <record id="tax_report_box13" model="account.tax.report.line">
+        <field name="name">[BOX 13] Credit adjustments from your calculation sheet</field>
+        <field name="parent_id" ref="tax_report_purchases_and_expenses"/>
+        <field name="sequence" eval="3"/>
+        <field name="country_id" ref="base.nz"/>
+    </record>
+
+    <record id="tax_report_box14" model="account.tax.report.line">
+        <field name="name">[BOX 14] Total GST credit for purchases and expenses</field>
+        <field name="parent_id" ref="tax_report_purchases_and_expenses"/>
+        <field name="sequence" eval="4"/>
+        <field name="formula">((BOX11 * 3)/23)</field>
+        <field name="country_id" ref="base.nz"/>
+    </record>
+
+    <record id="tax_report_box15" model="account.tax.report.line">
+        <field name="name">[BOX 15]  Difference between BOX10 and BOX14</field>
+        <field name="sequence" eval="3"/>
+        <field name="formula">((((BOX5 - BOX6) * 3)/23) + BOX9) - ((BOX11 * 3)/23)</field>
```

Comment:
It would be better to se the difference BOX10 - BOX14 (by also giving them codes)